core/src/test/scala/integration/kafka/api/CustomQuotaCallbackTest.scala

```diff
@@ -87,11 +90,25 @@ class CustomQuotaCallbackTest extends IntegrationTestHarness with SaslSetup {
 
   override def configureSecurityBeforeServersStart(testInfo: TestInfo): Unit = {
     super.configureSecurityBeforeServersStart(testInfo)
+  }
```

Comment:
Do we still need to keep `configureSecurityBeforeServersStart()` if all it does is call the `super` implementation?